<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OLMMIPV" ignored="0" name=" Chrome Test" tests="5" failures="1" timestamp="2023-05-12T12:03:03 IST" time="42.88" errors="0">
  <testcase name="@BeforeMethod Setup" time="22.29" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <failure type="org.openqa.selenium.TimeoutException" message="timeout: Timed out receiving message from renderer: 19.583
  (Session info: chrome=112.0.5615.139)
Build info: version: &amp;apos;4.8.3&amp;apos;, revision: &amp;apos;e5e76298c3&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_202&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [83c240481c8b3a535f3959c04d6e55bc, get {url=https://qalegend.com/billing/public/login}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.139, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: C:\Users\Anusree\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:57047}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:57047/devtoo..., se:cdpVersion: 112.0.5615.139, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 83c240481c8b3a535f3959c04d6e55bc">
      <![CDATA[org.openqa.selenium.TimeoutException: timeout: Timed out receiving message from renderer: 19.583
  (Session info: chrome=112.0.5615.139)
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [83c240481c8b3a535f3959c04d6e55bc, get {url=https://qalegend.com/billing/public/login}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 112.0.5615.139, chrome: {chromedriverVersion: 112.0.5615.49 (bd2a7bcb881c..., userDataDir: C:\Users\Anusree\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:57047}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:57047/devtoo..., se:cdpVersion: 112.0.5615.139, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 83c240481c8b3a535f3959c04d6e55bc
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at com.QALegend.qa.Base.TestBase.Initialization(TestBase.java:66)
at com.QALegend.qa.Testcases.LoginPageTestcases.Setup(LoginPageTestcases.java:34)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod Setup -->
  <testcase name="@AfterMethod quit" time="-1.683873183653E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod Setup" time="-1.683873183656E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @BeforeMethod Setup -->
  <testcase name="@AfterMethod quit" time="-1.683873183665E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod Setup" time="-1.68387318367E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @BeforeMethod Setup -->
  <testcase name="@AfterMethod quit" time="-1.683873183687E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="@BeforeMethod Setup" time="-1.683873183702E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @BeforeMethod Setup -->
  <testcase name="@AfterMethod quit" time="-1.683873183712E9" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- @AfterMethod quit -->
  <testcase name="LoginwithValidCredentialsTC1" time="3.665" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <failure type="java.lang.AssertionError" message="expected [https://qalegend.com/billing/public/home] but found [https://qalegend.com/billing/public/login]">
      <![CDATA[java.lang.AssertionError: expected [https://qalegend.com/billing/public/home] but found [https://qalegend.com/billing/public/login]
at com.QALegend.qa.Testcases.LoginPageTestcases.LoginwithValidCredentialsTC1(LoginPageTestcases.java:43)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- LoginwithValidCredentialsTC1 -->
  <testcase name="LoginwithInValidCredentialsTC2" time="0.0" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- LoginwithInValidCredentialsTC2 -->
  <testcase name="ValidusernameinvalidpwdTC3" time="0.0" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- ValidusernameinvalidpwdTC3 -->
  <testcase name="InvalidusernamevalidpwdTC4" time="0.0" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- InvalidusernamevalidpwdTC4 -->
  <testcase name="ForgotpwdTC5" time="0.0" classname="com.QALegend.qa.Testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- ForgotpwdTC5 -->
</testsuite> <!--  Chrome Test -->
